docs/src/operations/requirements.md

```diff
@@ -19,8 +19,8 @@ transaction for each block the validator agrees with, which can cost up to
 The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
 
 - CPU
-  - 12 cores / 24 threads, or more
-  - 2.8GHz base clock speed, or faster
+  - 16 cores / 24 threads, or more
+  - 3.0 GHz base clock speed, or faster
```

Comment:
I poked around our fleet a bit, we do have a decent number of machines running with this processor:
https://www.amd.com/en/products/cpu/amd-epyc-7402p

> Base Clock 2.8GHz

So, I'm a little hesitant to change this at the moment

Reply:
might actually go down if we can get mm and vfs out of the way